code/game/turfs/turf.dm

```diff
@@ -928,3 +928,12 @@ GLOBAL_LIST_INIT(blacklisted_automated_baseturfs, typecacheof(list(
 	underlay_appearance.icon_state = icon_state
 	underlay_appearance.dir = adjacency_dir
 	return TRUE
+
+///Are we able to teleport to this turf using in game teleport mechanics
+/turf/proc/can_teleport_here()
+	if(density)
+		return FALSE
+	//var/signalreturn = SEND_SIGNAL(src, COMSIG_TURF_TELEPORT_CHECK)
```

Comment:
Commented code stinky

Reply:
whelp, local maint unable to proof check their own code.